staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -395,6 +395,14 @@ type fsStatusUpdate struct {
 	oldValue   flowcontrol.FlowSchemaCondition
 }
 
+type objectForDigestConfigObjectsStatus struct {
```

Comment:
How about a name that describes what this is, rather than where it is used?  Perhaps something like FlowSchemaOnlyStatusCondition for the outer type, FlowSchemaStatusOnlyCondition for the inner type?